modules/rules_java/7.0.5/presubmit.yml

```
@@ -0,0 +1,9 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+tasks:
+  verify_build_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    bazel_version: 6.4.0rc5
```

```suggestion
    bazel: 6.4.0rc5
```
See https://github.com/bazelbuild/bazel-central-registry/blob/356e1826fb3114e0fde241a57ca2a6624a7ceeb6/modules/with_cfg.bzl/0.1.0/presubmit.yml#L15

Sorry about the mixup!

---

Sigh.